### ISSUE TITLE
Make ansibot less chatty

### ISSUE THREAD

**Issue by dagwieers** [ansible/ansibullbot#1081]
I understand the need for presenting more information to (new) users, but having multiple comments added in close succession is a bit too much. Each comment induces another mail.

For a typical new PR/issue we get:
1. One comment to thank the user -- *Can we do this only for new contributors ?*
2. One comment to cc related authors
3. One coment to inform them of an existing WG

Each of these comments link to the ansibot help documentation.

Could we make ansibot smarter so it combines this information in a single comment ? Especially 1. and 3. seem a good introductionary blob, but I would also include the other authors by replacing **cc @author1 @author2** with **The following people are now notified of your issue/PR: @author1 @author2** (we could even make a distinction between maintainers and people notified in that message if that is deemed useful)

**Comment by webknjaz** [comment]
I think that everyone deserves to be thanked.

As of other points, I 100% agree with those.

**Comment by gundalow** [comment]
Hi,
One thing that was discussed at Contributors Summit in Austin was that new issues/PR should get a **single** commented added along the forms of:


```

## Info for $AUTHOR
@person, thank you for this PR

If you are new to Contributing to Ansible then $LINK describes the process

If you'd like to add test see $LINK

If you get stuck, feel free to join us in #ansible-devel see $IRC_DOCS_LINK
## Info for Maintainers
The following people are listed as maintainers: $MAINTAINERS, can you please review this

## Working Group

This ISSUE/PR related to the following working groups...
* wg1
* wg2
LINK TO STUFF

## Info for other reviews

Testing of PRs is the best type of feedback, to download this patch do
$COMMAND_FOR_PSEUDO_BRANCH
see $DOCS_LINK

Please add comments saying if you've tested this and how it works

```

**Comment by dagwieers** [comment]
@gundalow Perfect. I also wonder whether we can make this shorter for regular contributors. They don't need to be forwarded to working groups or being lectured about the process. It just adds more noise to the thread IMO. Less is more.

@webknjaz Being thanked by a bot doesn't feel very special to me. I don't think it's very flattery, but I can imagine new contributors are more affected by it.

**Comment by ckotte** [comment]
I prefer one email for a PR instead of three. It's also annoying that you get a single email for every single code comment, but I assume this is a Github thing and not related to ansibot.

**Comment by webknjaz** [comment]
@ckotte yes, that's github's setting. You can configure that in your own notification settings.

**Comment by dagwieers** [comment]
Can we please get the "Thank you" message disabled until we have fixed this ?
It's really annoying to get additional noise for no good reason.

**Comment by gundalow** [comment]
@dagwieers yup, will do

**Comment by gundalow** [comment]
"Thanks" message has been removed
